### PR TITLE
avoid "by zero"  exception when all request are error and another class not found exception fix

### PR DESCRIPTION
**Pull request by candyleer**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.
1. when all request are error ,it will occur "by zero exception" thus web-ui cannot show the right data
2. class not found exception when agent started,may be a spell  error 😆
``` 
Caused by: java.lang.ClassNotFoundException: Can't find Jorg.apache.skywalking.apm.plugin.jedis.v2.edisClusterConstructorWithListHostAndPortArgInterceptor
	at org.apache.skywalking.apm.agent.core.plugin.loader.AgentClassLoader.findClass(AgentClassLoader.java:119)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.apache.skywalking.apm.agent.core.plugin.loader.InterceptorInstanceLoader.load(InterceptorInstanceLoader.java:77)
	at org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.ConstructorInter.<init>(ConstructorInter.java:51)
	... 67 more
```
- How to fix?
1. add some "if else" to avoid that
2. spell fix
___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: apm-collector/apm-collector-ui/collector-ui-jetty-provider/src/main/java/org/apache/skywalking/apm/collector/ui/service/ServiceTopologyService.java [hunk 1]
```diff
@@ -86,7 +86,11 @@ public Topology getServiceTopology(Step step, int serviceId, long startTimeBucke
             Call call = new Call();
             call.setSource(referenceMetric.getSource());
             call.setTarget(referenceMetric.getTarget());
-            call.setAvgResponseTime((referenceMetric.getDurations() - referenceMetric.getErrorDurations()) / (referenceMetric.getCalls() - referenceMetric.getErrorCalls()));
+            if (referenceMetric.getCalls() == referenceMetric.getErrorCalls()) {
+                call.setAvgResponseTime(referenceMetric.getDurations() / referenceMetric.getCalls());
```
Comment (wu-sheng) [inline apm-collector/apm-collector-ui/collector-ui-jetty-provider/src/main/java/org/apache/skywalking/apm/collector/ui/service/ServiceTopologyService.java]: This process way has conflicts with other. If all requests are error, the avg response time should be 0.

Comment (candyleer) [reply apm-collector/apm-collector-ui/collector-ui-jetty-provider/src/main/java/org/apache/skywalking/apm/collector/ui/service/ServiceTopologyService.java]: fixed.avg time has been set to 0

**Comment by candyleer** [comment]
```
private static final String CONSTRUCTOR_WITH_LIST_HOSTANDPORT_ARG_INTERCEPT_CLASS = "Jorg.apache.skywalking.apm.plugin.jedis.v2.edisClusterConstructorWithListHostAndPortArgInterceptor";
```
bug or on purpose?

**Comment by wu-sheng** [comment]
@candyleer I assume it is a bug. Please help to fix this in another pr.

**Comment by candyleer** [comment]
ok , i will fix this in another pr

**Pull request closed without merging**